demo/api/src/news/generated/news.crud.resolver.ts

```diff
@@ -35,10 +37,18 @@ export class NewsCrudResolver {
     }
 
     @Query(() => PaginatedNews)
-    async newsList(@Args() { scope, search, filter, sort, offset, limit }: NewsListArgs): Promise<PaginatedNews> {
+    async newsList(@Args() { scope, search, filter, sort, offset, limit }: NewsListArgs, @Info() info: GraphQLResolveInfo): Promise<PaginatedNews> {
         const where = this.newsService.getFindCondition({ search, filter });
         where.scope = scope;
-        const options: FindOptions<News> = { offset, limit };
+
+        const fields = extractGraphqlFields(info, { root: "nodes" });
+        const populate: string[] = [];
+        if (fields.includes("comments")) {
+            populate.push("comments");
+        }
```

Comment:
Why have both the populate and the field resolver that calls `loadItems`? Shouldn't it be possible to remove the field resolver, as the data is loaded already anyway?

Reply:
Haven't thought about that! 

I thought this may lead to relations being loaded twice in get-multiple-items resolvers, but it seems that `loadItems` checks whether items have been loaded already: https://github.com/mikro-orm/mikro-orm/blob/master/packages/core/src/entity/Collection.ts#L49